content/en/docs/ops/common-problems/upgrade-issues/index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Upgrade Problems
+description: Resolve common problems with Istio upgrades.
+weight: 40
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
+---
+
+## `EnvoyFilter` migration
```

Comment:
```suggestion
### `EnvoyFilter` migration
```

Reply:
No, this is right because these items are just for EnvoyFilter upgrade issues.